src/Microsoft.DotNet.ProjectModel/Graph/LockFileReader.cs

```diff
@@ -17,7 +18,17 @@ public static class LockFileReader
     {
         public static LockFile Read(string lockFilePath)
```

Comment:
There should be no retry by default. Only the project model server should be doing the retry.